src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -75,10 +73,20 @@ MusicBrainzRecordingsTask::MusicBrainzRecordingsTask(
                   networkAccessManager,
                   parent),
           m_queuedRecordingIds(recordingIds),
-          m_timer(0),
           m_parentTimeoutMillis(0) {
-    m_timer.start();
     musicbrainz::registerMetaTypesOnce();
+    // Some of the tracks has loads of RecordingIds.
+    // This can cause hitting the rate limits of MusicBrainz.
+    // According to the MusicBrainz API Doc: https://musicbrainz.org/doc/MusicBrainz_API/Rate_Limiting
+    // The rate limit should be one query in a second.
+    // Related Bug: https://bugs.launchpad.net/mixxx/+bug/1983204
+    // In order to not hit the rate limits and respect their rate limiting rule.
+    // We are going to delay every request by one second.
+    m_timer.start(m_DelayDurationMilis);
```

Comment:
Start the QTimer in place of the original sleep() 
With the same calculation from te other timer. 
Maybe call one inhibitTimer and the other RequestTimerr or such.

Reply:
Done :+1: